internal/service/verify/resource_voice_phrase_content_test.go

```diff
@@ -0,0 +1,223 @@
+package verify_test
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/pingidentity/terraform-provider-pingone/internal/acctest"
+	validation "github.com/pingidentity/terraform-provider-pingone/internal/verify"
+)
+
+func testAccCheckVoicePhraseContentsDestroy(s *terraform.State) error {
+	var ctx = context.Background()
+
+	p1Client, err := acctest.TestClient(ctx)
+
+	if err != nil {
+		return err
+	}
+
+	apiClient := p1Client.API.VerifyAPIClient
+
+	mgmtApiClient := p1Client.API.ManagementAPIClient
+
+	for _, rs := range s.RootModule().Resources {
+		if rs.Type != "pingone_voice_phrase_content" {
+			continue
+		}
+
+		_, rEnv, err := mgmtApiClient.EnvironmentsApi.ReadOneEnvironment(ctx, rs.Primary.Attributes["environment_id"]).Execute()
+
+		if err != nil {
+
+			if rEnv == nil {
+				return fmt.Errorf("Response object does not exist and no error detected")
+			}
+
+			if rEnv.StatusCode == 404 {
+				continue
+			}
+
+			return err
+		}
+
+		body, r, err := apiClient.VoicePhraseContentsApi.ReadOneVoicePhraseContent(ctx, rs.Primary.Attributes["environment_id"], rs.Primary.Attributes["voice_phrase_id"], rs.Primary.Attributes["id"]).Execute()
+
+		if err != nil {
+
+			if r == nil {
+				return fmt.Errorf("Response object does not exist and no error detected")
+			}
+
+			if r.StatusCode == 404 {
+				continue
+			}
+
+			tflog.Error(ctx, fmt.Sprintf("Error: %v", body))
+			return err
+		}
+
+		return fmt.Errorf("PingOne Voice Phrase Content %s still exists", rs.Primary.ID)
+	}
+
+	return nil
+}
+
```

Comment:
I recently added new tests for correct handling for resource removal outside of Terraform.  This resource will need the new test added.  Example:

https://github.com/pingidentity/terraform-provider-pingone/blob/e3183b2c330670ad26c307f8875786c0c8b6fe02/internal/service/sso/resource_application_resource_grant_test.go#L69-L132